src/pyrtcm/rtcmmessage.py

```diff
@@ -259,11 +259,17 @@ def _set_attribute_single(
                 # populate NSAT and NCELL mapping dictionaries
                 self._getsatcellmaps()
 
-        # add special coefficient attributes for message 4076_201
-        if anam == "IDF038":
+        # add special coefficient attributes for message 4076_201 and 1264
+        special_cases = {
+            "IDF038": ("IDF037", "IDF038"),  # message 4076_201
+            "DF475": ("DF476", "DF475"),  # message 1264

```

Comment:
Should this not be 

```python
special_cases = {
            "IDF038": ("IDF037", "IDF038"),  # message 4076_201
            "DF475": ("DF474", "DF475"),  # message 1264
```
i.e. DF474 (Spherical Harmonics Degree) rather than DF476 (Spherical Harmonic Coefficient C)

Reply:
Good eye, thanks.